docs/src/modules/components/ThemeContext.js

```diff
@@ -48,12 +48,32 @@ export function Provider(props) {
         throw new Error(`Unrecognized type ${action.type}`);
     }
   }, themeInitialOptions);
-  const { direction, paletteColors, paletteType } = themeOptions;
+
+  const prefersDarkMode = useMediaQuery('prefers-color-scheme: dark');
```

Comment:
```suggestion
  const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
```

Reply:
Going with the `@media` prefix to make it more obvious why those are needed.